order/rpc_parse.go

```diff
@@ -89,7 +90,9 @@ func ParseRPCOrder(version, leaseDuration uint32,
 		return nil, errors.New("min units match must be greater than 0")
 
 	// The min units match must not exceed the total order units.
-	case details.MinUnitsMatch > uint32(kit.Units):
+	case kit.AuctionType != BTCOutboundLiquidity &&
+		details.MinUnitsMatch > uint32(kit.Units):
```

Comment:
I think the previous condition was valid even if ```kit.AuctionType == BTCOutboundLiquidity```. But I might be misunderstanding the notion of units as they pertain to the outbound liquidity auction type.